packages/cadl-rlc-test/cadl-output/src/responses.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpResponse } from "@azure-rest/core-client";
+import { ErrorOutput, WidgetOutput } from "./outputModels";
+
+/** Ok */
+export interface WidgetServiceList200Response extends HttpResponse {
+    status: "200";
+}
+
+export interface WidgetServiceListdefaultResponse extends HttpResponse {
```

Comment:
Can we fix the lowercase `d` in default? I know it was like this before, but I think we should fix it